tests/index.html

```diff
@@ -23,15 +23,6 @@
       (function() {
         window.EmberENV = window.EmberENV || {};
 
-        // Determine edition, and apply correct EmberENV settings
-        var edition = QUnit.urlParams.edition || 'octane';
-
-        if (edition === 'octane') {
-          EmberENV._TEMPLATE_ONLY_GLIMMER_COMPONENTS = true;
```

Comment:
This defaults to false https://github.com/emberjs/ember.js/blob/main/packages/%40ember/-internals/environment/lib/env.ts#L97

I suspect the others also default to their "classic" values. There are other tests that set the optional flags one way or another and probably what we need to do is remove the optional features we no longer support, but at the very least, our tests should keep the octane version of tests.

Reply:
Good catch. I'm pretty sure it's impossible for a real app to set these to the non-octane values because of the assertions in lib/index.js. So we can probably just rip these flags out and hard-code the octane behavior.

Reply:
Yeah I think it is just missed cleanup

Reply:
Actually looks like there's no assertion on async observers

Reply:
We should deprecate synchronous observers. Whether by 6 or by 7 I don't have a strong opinion, it would depend on how prevalent the community dependence still is.